pages/measurement.js

```diff
@@ -50,7 +50,7 @@ export default class Measurement extends React.Component {
       initialProps['isAnomaly'] = results[0].anomaly
       initialProps['isFailure'] = results[0].failure
       initialProps['isConfirmed'] = results[0].confirmed
-
+      initialProps['message'] = results[0].scores.msg || null
```

Comment:
I think you should probably inject into the `measurement` the whole `scores` dictionary and we can maybe for the time being, for debugging purposes, just log it to the console.

Eventually I think we would like to show it somewhere in the measurement page itself, probably somewhere close to the heading.

The `scores` field is what you are going to be using in order to extract the WhatsApp related that are calculated by the fastpath, see: https://github.com/ooni/pipeline/pull/297/files#diff-88c24525382330f3fc745dfea1834e78R765.

Basically you will get a data structure like this:
```
"scores": {
      "analysis": {
        "registration_server_accessible": true,
        "whatsapp_web_accessible": false,
        "whatsapp_endpoints_accessible": true,
    }
}
```

Which map to what you have in the WhatsApp test as: https://github.com/ooni/explorer/blob/master/components/measurement/nettests/WhatsApp.js#L30

Reply:
Done